<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1435</width>
    <height>888</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PHP Extension Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>1111</width>
      <height>561</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Source File</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_cFile">
        <property name="text">
         <string>Current C File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textEdit_cFile">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Header File</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_hFile">
        <property name="text">
         <string>Current H File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textEdit_hFile">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>config.m4</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Please note: Any changes to this document are saved strait away.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_config_m4">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>config.w32</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Please note: Any changes to this document are saved strait away.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_config_w32">
        <property name="font">
         <font>
          <family>Courier</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_compileExtension">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>770</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile Extension</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_compileType">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>730</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Shared Object</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Static</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_path_phpSDKBase">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>630</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_compileBase">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>660</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile Base Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_path_extBase">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>690</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Extension Base:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>600</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Compiler Properties</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>730</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Debug</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Release</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Extension Files:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>108</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User-Space Functions:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_extensionFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_phpFE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Others:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>580</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>200</width>
      <height>190</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>INI ENTRIES</string>
     </attribute>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>ARG INFO</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Helper Functions</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_compile">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>580</y>
      <width>771</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Compiler Output</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_compile_out">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_compileClearOutput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Compile Output</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Command Line</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1435</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Extension"/>
    <addaction name="actionCreate_Extension"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Current_Document"/>
    <addaction name="actionSave_All_Documents"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuExtension">
    <property name="title">
     <string>Extension</string>
    </property>
    <addaction name="actionExtension_Properties"/>
    <addaction name="actionCreate_Test"/>
    <addaction name="actionCompile_Extension"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuExtension"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Extension">
   <property name="text">
    <string>Load Extension...</string>
   </property>
  </action>
  <action name="actionCreate_Extension">
   <property name="text">
    <string>Create  Extension...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionGo_to_PHP_Function">
   <property name="text">
    <string>Go to PHP Function</string>
   </property>
  </action>
  <action name="actionCreate_Test">
   <property name="text">
    <string>Create Test</string>
   </property>
  </action>
  <action name="actionCompile_Extension">
   <property name="text">
    <string>Compile Extension...</string>
   </property>
  </action>
  <action name="actionExtension_Properties">
   <property name="text">
    <string>Extension Properties...</string>
   </property>
  </action>
  <action name="actionSave_Current_Document">
   <property name="text">
    <string>&amp;Save Current Document</string>
   </property>
  </action>
  <action name="actionSave_All_Documents">
   <property name="text">
    <string>Save All Documents</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
